app/routes/events/utils.ts

```diff
@@ -32,16 +32,16 @@ export const eventTypeToString = (eventType: EventType): string => {
   return EVENT_CONSTANTS[eventType] || EVENT_CONSTANTS['other'];
 };
 // Colors for different event types
-const COLOR_CONSTANTS: Record<EventType, string> = {
+export const COLOR_CONSTANTS: Record<EventType, string> = {
   company_presentation: '#A1C34A',
-  lunch_presentation: '#A1C34A',
+  lunch_presentation: '#98f942',
   alternative_presentation: '#8A2BE2',
   course: '#52B0EC',
   breakfast_talk: '#86D1D0',
   party: '#FCD748',
-  social: 'var(--color-event-red)',
```

Comment:
I'm sorry, but I don't really like the pink color. Also, the new green for `lunch_presentation` it a bit too bright for my taste.

<img width="909" alt="image" src="https://github.com/webkom/lego-webapp/assets/69514187/f81a9bcc-59ec-4588-866d-eea120873fa3">

<img width="407" alt="image" src="https://github.com/webkom/lego-webapp/assets/69514187/5983c42e-c4a8-4273-b183-1d5199b2e26d">